### PR TITLE
WIP: Convert zone pivots config to YAML

### PR DESCRIPTION
**Pull request by scottaddie**

### CODE REVIEW

File: aspnetcore/blazor/tooling.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ By [Daniel Roth](https://github.com/danroth27) and [Luke Latham](https://github.
 
 ::: zone pivot="os-windows"
 
-1. Install the latest version of [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/) with the **ASP.NET and web development** workload.
+1. Install the latest version of [Visual Studio 2019](https://visualstudio.microsoft.com/downloads) with the **ASP.NET and web development** workload.
```
Comment (guardrex) [inline aspnetcore/blazor/tooling.md:19]: This 301 redirects to the `/` URL.

File: aspnetcore/blazor/tooling.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ There's no centralized way to trust a certificate on Linux. Typically, one of th
 * Trust all self-signed certificates for `localhost`.
 * Add the certificate to the list of trusted certificates in the browser.
 
-For more information, see the guidance provided by your browser and Linux distro.
+For more information, see the guidance provided by your browser and Linux distribution.
```
Comment (guardrex) [inline aspnetcore/blazor/tooling.md:77]: There are four more of these across the repo.

"distro" is a valid abbreviation of "distribution." It's used *everywhere* by just about *everyone*. There's no confusion. Maintainers use it to describe their own products. Why are we trying to force the word "distribution" in these docs?

Comment (guardrex) [reply aspnetcore/blazor/tooling.md:77]: Nevermind ... I was just curious.

Comment (scottaddie) [reply aspnetcore/blazor/tooling.md:77]: You're correct that most folks would say "distro", including myself. The problem is localization. This term would be considered slang/jargon.

Comment (guardrex) [reply aspnetcore/blazor/tooling.md:77]: Gotcha ... that makes sense. Thanks.

**Comment by scottaddie** [comment]
Replaced with https://github.com/dotnet/AspNetCore.Docs/pull/19175.

File: aspnetcore/blazor/tooling.md (hunk 2 above)
Comment (scottaddie) [reply aspnetcore/blazor/tooling.md:77]: I agree. I'll replace these with another PR right now.

**Pull request closed without merging**